contracts/token/Minter.sol

```diff
@@ -95,14 +110,35 @@ contract Minter is Manager, IMinter {
      * @dev Set the reward token amounts for the round. Only callable by the RoundsManager
      */
     function setCurrentRewardTokens() external onlyRoundsManager whenSystemNotPaused returns (bool) {
+        setInflation();
+
         currentMintableTokens = mintedTokensPerRound();
         currentMintedTokens = 0;
-        currentRedistributableTokens = redistributableTokensPerRound();
+        currentRedistributableTokens = redistributionPool;
         currentRedistributedTokens = 0;
 
         return true;
     }
 
+    /*
+     * @dev Set inflation based upon the current bonding rate
+     */
+    function setInflation() internal {
+        uint256 currentBondingRate = (bondingManager().getTotalBonded() * PERC_DIVISOR) / livepeerToken().totalSupply();
+
+        uint256 newInflation = inflation;
+
+        if (currentBondingRate < targetBondingRate) {
+            // Bonding rate is below the target - increase inflation
+            newInflation = newInflation.add(inflationChange);
+        } else if (currentBondingRate > targetBondingRate) {
+            // Bonding rate is above the target - decrease inflation
+            newInflation = newInflation.sub(inflationChange);
+        }
```

Comment:
Should we add a check here to make sure inflation stays between some range?  (at least above 0?)

Reply:
Discussed previously offline with @dob - adding upper/lower bounds might make sense, but they also introduce additional parameters that we do not know the right values for. Additionally, there might be some benefit in allowing the inflation rate to float freely and allow  participation by delegators with various incentives and liquidity preferences adjust the rate without us trying to prescribe what we believe to be the right bounds. An example could be if the inflation does end up starting to increase out of control, anyone with belief in the network value can be play a sort of "lender of last resort" role (ala a central bank except anyone can play this role) and bond to increase the participation rate to the target rate and stop the inflation growth. That being said, I think this is just an initial implementation to add the functionality and we can definitely do more research and discuss better models